eahub/templates/components/feedback.html

```diff
@@ -0,0 +1,50 @@
+{% block feedback %}
+{% load static %}
+
+  
+  <!-- Button trigger modal -->
+  <button type="button" class="btn feedbackbtn" data-bs-toggle="modal" data-bs-target="#feedbackModal">
+    Feedback?
```

Comment:
When I click on the button, nothing happens (Desktop, Firefox)

Reply:
That's odd... It works for me and I tested in Safari, Chrome and Firefox on desktop and small screens:

<img width="463" alt="image" src="https://user-images.githubusercontent.com/24226083/112276626-dd32f300-8c78-11eb-9e71-92c5cc3de372.png">

Is this the correct way to link the javascript?
`<script src="{% static 'components/feedback.js' %}" type="text/javascript"></script>`

Reply:
Hm, it's not working for me. Maybe you didn't push your latest commits?

Reply:
figured it out: had to rerun ```npm ci```.